create/faq.md

```diff
@@ -189,4 +176,45 @@ See our [contact page](/learn/what_is_meilisearch/contact.md).
 
 ## I have just updated MeiliSearch, and I am getting an error: "Cannot open database, expected MeiliSearch engine version..."
 
-MeiliSearch minor versions are not compatible with each other because the way we represent data internally changes with the addition of new breaking features. To fix this, it suffices to delete your database folder (`data.ms` by default) and re-index your documents with the new version.
+MeiliSearch minor versions are not compatible with each other because the way we represent data internally changes with the addition of new breaking features. To fix this, it suffices to delete your database folder (`data.ms` by default) and re-index your documents with the new version. See [updating MeiliSearch](/reference/features/installation.md#updating-meilisearch) for more information.
+
+## What are the recommended requirements for hosting a MeiliSearch instance?
+
+**Short answer**: It's impossible to say. We recommend [experimenting with RAM availability and database size](#how-does-hardware-affect-search-speed) to find the hardware + relevancy setup that works best for your dataset.
+
+**Long answer**: The disk size and RAM usage of your MeiliSearch database can vary widely. The following factors have a great effect on space and memory use:
```

Comment:
Are they listed in DESC order of impact? If so, we should say it. If not, we should list them in order if possible. If not possible, we should mention it.

Reply:
Good point. I'll ask @qdequele about it in the next docs sync meeting. For now, I think we should forge ahead and merge this PR.